Cartfile

```diff
@@ -1,2 +1,2 @@
 github "raphaelmor/Polyline" ~> 5.0
-github "mapbox/turf-swift" ~> 2.6.1
+github "mapbox/turf-swift" ~> 2.7.0
```

Comment:
Should be fine to bump to 2.8?

Reply:
There are Maps versions which use exactly 2.7.0, so we need a compatible version.